server/webapp/WEB-INF/rails/webpack/views/components/expand_collapsible/index.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import "../../global/theme";
+@import "../../global/measures";
+@import "../../global/common";
+
+.collapse {
+  background:    $page-header-bg;
+  position:      relative;
+  border:        1px solid $border-color;
+  border-radius: $global-border-radius;
+  margin-bottom: 10px;
+  &.collapsed {
+    background: $page-header-bg;
+  }
+  &.expanded {
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.21);
+    .c-collapse_header {
```

Comment:
FYI: we're moving to `camelCase` classnames. Talk to @naveenbhaskar and @akshaydewan on how to best name classes now.

Reply:
@ketan -- This was just a copy paste from the existing code. @naveenbhaskar is going to work on the styles bit and probably will remove this class.